docs/references/nextjs/clerk-middleware.mdx

```diff
@@ -295,4 +295,20 @@ The `clerkMiddleware()` function accepts an optional object. The following optio
 | `signInUrl?`                       | `string`             | An alternative sign in URL.                                                                                                                                                                                                                                                                                                                     |
 | `signUpUrl?`                       | `string`             | An alternative sign up URL.                                                                                                                                                                                                                                                                                                                     |
 | `publishableKey` | `string` | The Clerk publishable key for your instance. This can be found in your Clerk Dashboard on the **[API Keys](https://dashboard.clerk.com/last-active?path=api-keys)** page.|
-| `secretKey?` | `string` | The Clerk secret key for your instance. This can be found in your Clerk Dashboard on the **[API Keys](https://dashboard.clerk.com/last-active?path=api-keys)** page.  |
+| `secretKey?` | `string` | The Clerk secret key for your instance. This can be found in your Clerk Dashboard on the **[API Keys](https://dashboard.clerk.com/last-active?path=api-keys)** page. The `CLERK_ENCRYPTION_KEY` environment variable must be  when providing `secretKey` as an option, refer to [Dynamic keys](#dynamic-keys). |
```

Comment:
```suggestion
| `secretKey?` | `string` | The Clerk secret key for your instance. This can be found in your Clerk Dashboard on the **[API Keys](https://dashboard.clerk.com/last-active?path=api-keys)** page. The `CLERK_ENCRYPTION_KEY` environment variable must be set when providing `secretKey` as an option, refer to [Dynamic keys](#dynamic-keys). |
```